src/renderer/shared/transactions/createTransactionConfirmStore.ts

```diff
@@ -16,7 +16,7 @@ import { walletUtils } from '@/entities/wallet';
 export type TxConfirmInfo = {
   id?: number;
   initiator: AnyAccount;
-  signatory: AnyAccount;
+  signatory: AnyAccount | null;
```

Comment:
That's wrong behavior, signatory should always exist, but most of the time it's overlapping initiator